test/integration/smoke/test_activate_template.py

```diff
@@ -0,0 +1,241 @@
+#!/usr/bin/env python
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from marvin.lib.base import (Account,
+                             ServiceOffering,
+                             VirtualMachine,
+                             DiskOffering,
+                             Template,
+                             Volume,
+                             Zone)
+from marvin.lib.common import (get_domain,
+                               get_zone,
+                               get_template)
+from marvin.cloudstackTestCase import cloudstackTestCase, unittest
+from nose.plugins.attrib import attr
+import time
+from marvin.cloudstackAPI import (registerTemplate, seedSystemVMTemplate, listTemplates, listOsTypes, activateSystemVMTemplate, createZone, listConfigurations, getSystemVMTemplateDefaultUrl, deleteTemplate)
+from marvin.lib.utils import cleanup_resources
+
+class TestActivateTemplate(cloudstackTestCase):
```

Comment:
I see several hardcoded values here, but as these are in a test they might be ok. No blokker as they will become clear at the very next occasion/upgrade, if they harm in any way.